events.md

```diff
@@ -7,4 +7,4 @@ navigation_order: 1
 
 These are the planned events for Monthly Music Hackathon. Check out each one for more details and to RSVP.
 
-For past events, check in the [Event Archive](/events/archive.html).
```

Comment:
I think the main thing that needs to be done is add a trailing slash to the permalink in the event_archive.md